sdk/component/storer.go

```diff
@@ -1,20 +1,26 @@
 package component
 
-import "github.com/smithy-security/smithy/sdk/component/internal/storer/local"
+import (
+	"fmt"
+
+	"github.com/smithy-security/smithy/sdk/component/internal/storer/local/sqlite"
+)
 
 type storeType string
 
-const (
-	storeTypeTest  storeType = "test"
-	storeTypeLocal storeType = "local"
-)
+const storeTypeLocal storeType = "local"
```

Comment:
I think this should be an enum so that we can ensure the validity of the options. we are using the https://github.com/abice/go-enum in other places to make sure that the enumeration is actually useful and safe

Reply:
the string enums in go are a fotogun, for example you could do `storeType("bla")` and the compiler has no issue with this but the wrong value would be allowed to progress quite a bit throughout the code before (hopefully) a switch or an if will figure out that the value is incorrect. The problem is that when writing code that uses enums you make the (reasonable) assumption that there are only specific values that you deal with, not any value out there. But in Golang that's not the case, in contrast to pretty much every other programming language out there.